handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/filter/ZoneAwareServerListFilterExt.java

```diff
@@ -20,33 +20,53 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import org.apache.servicecomb.loadbalance.CseServer;
+import org.apache.servicecomb.core.Invocation;
+import org.apache.servicecomb.loadbalance.LoadBalancer;
+import org.apache.servicecomb.loadbalance.ServiceCombServer;
 import org.apache.servicecomb.loadbalance.ServerListFilterExt;
 import org.apache.servicecomb.serviceregistry.RegistryUtils;
 import org.apache.servicecomb.serviceregistry.api.registry.MicroserviceInstance;
 
+import com.netflix.config.DynamicPropertyFactory;
 import com.netflix.loadbalancer.Server;
 
 public class ZoneAwareServerListFilterExt implements ServerListFilterExt {
```

Comment:
is it possible that move old filters to DiscoveryFilter?
so that we can cache filter result in most scenes